### DISCUSSION TITLE
一张图片有多种语言的文字如何识别

### DISCUSSION THREAD

**Discussion by cxyz1** [PaddlePaddle/PaddleOCR#12967] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by terrencew** [comment]
检测时增加语言种类分类或者在之后增加一个模型，用于语言分类

**Comment by cxyz1** [comment]
> 检测时增加语言种类分类或者在之后增加一个模型，用于语言分类
感谢回复！
1.我试了在检测之前增加语种识别，但是paddleOCR貌似只支持 （英文+其他语种）两种语言的检测，语言多了就不行了；
2.另外在之后增加一个模型用于语言分类是什么意思？
